test/functional.jl

```diff
@@ -513,3 +513,11 @@ let s = "Monkey 🙈🙊🙊"
     @test tf(2) == "Mo|nk|ey| 🙈|🙊🙊"
     @test tf(1) == "M|o|n|k|e|y| |🙈|🙊|🙊"
 end
+
+# Issue 4914
+let
```

Comment:
core is probably better for this. functional is for higher order functional programming